_data/simple-icons.json

```
@@ -6572,6 +6572,11 @@
             "source": "https://www.mixcloud.com/about",
             "guidelines": "https://www.mixcloud.com/about"
         },
+        {
+            "title": "Mixpanel",
+            "hex": "111111",
+            "source": "https://mixpanel.com"
```

I think the [press kit](https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9) should be provided as a source. Unfortunately, the URL isn't very beautiful and has a different domain. :man_shrugging:

Also, please add the URL to the press kit as a brand guidelines reference.

```suggestion
            "source": "https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9",
            "guidelines:" "https://www.notion.so/mxpnl/Mixpanel-Brand-Guidelines-538928c3ee9c4f2a9b330c5bfc38b5b9"
```